docs/index.html

```diff
@@ -82,6 +82,7 @@ <h2 class="subtitle">
 <footer class="footer">
   <div class="content has-text-centered">
     <a class="button is-primary kb-integrate kb-hide-on-install" href="https://github.com/apps/tdd1t">Integrate Now</a>
+    <a class="button is-primary" href="https://github.com/Kibibit/tdd1t">Documentation</a>
```

Comment:
This is not so much `Documentation` on how to install, as documentation on how to develop or just a link to the GitHub repository.

Also, it shouldn't take the main stage from the `integrate now` which will install the app.

- I suggest we either move this button under [line 94](https://github.com/Kibibit/tdd1t/pull/49/files#diff-eae600270579316beb8b8d9b9a5732d4R94) or as a link **in [line 94](https://github.com/Kibibit/tdd1t/pull/49/files#diff-eae600270579316beb8b8d9b9a5732d4R94) on the keyword `tdd1t`**). whatever you think is better.
- Rename the button to either `Repository`, `tdd1t on GitHub` or whatever describes the link better in your opinion (if you deside to keep the dedicated button)

If you have any documentation to add, add it to the homepage page since right now it's our main documentation on how to integrate and use until we'll implement #37 , which will allow us to generate the homepage with text included in the `README.MD` file.